Makefile

```diff
@@ -58,7 +58,7 @@ $(EBIN)/%.beam: $(ESRC)/%.erl
 
 $(TEST)/%.beam: $(TEST)/%.erl
 	@echo "T  ERLC $<"
-	@$(ERLC) $(EFLAGS) -o $(dir $<) $<
+	@$(ERLC) $(EFLAGS) +nowarn_export_all -o $(dir $<) $<
```

Comment:
I don't think it was failing the compilation, but then again I prefer to have it as clean as possible to not hide potential useful warnings.